extensions/panache/panache-common/deployment/src/main/java/io/quarkus/panache/common/deployment/PanacheEntityEnhancer.java

```diff
@@ -23,6 +23,7 @@
     public static final String JSON_PROPERTY_SIGNATURE = "L" + JSON_PROPERTY_BINARY_NAME + ";";
 
     public static final DotName JSON_IGNORE_DOT_NAME = DotName.createSimple("com.fasterxml.jackson.annotation.JsonIgnore");
+    public static final DotName JSON_PROPERTY_DOT_NAME = DotName.createSimple("com.fasterxml.jackson.annotation.JsonProperty");
```

Comment:
Is there a reason why we don't have the same mechanism for `@JsonbProperty`?

Reply:
Good question.

The reason is that these `@JsonProperty` shenanigans are only needed because of the `@XmlTransient` shenanigans - which Jsonb doesn't seem to care about at all.